arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/state/HubTraceSections.java

```diff
@@ -13,27 +13,29 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package net.consensys.linea.zktracer.module.hub;
+package net.consensys.linea.zktracer.module.hub.state;
 
 import java.util.ArrayList;
 import java.util.List;
 
 import com.google.common.base.Preconditions;
 import lombok.Getter;
 import lombok.experimental.Accessors;
+import net.consensys.linea.zktracer.module.hub.Trace;
 import net.consensys.linea.zktracer.module.hub.section.TraceSection;
 
 /**
- * Stores all the trace sections associated to the same transaction, stored in chronological order
- * of creation.
+ * Stores all the trace sections associated to one transaction, stored in chronological order of
+ * creation.
  */
+@Getter
 @Accessors(fluent = true)
-public class TxTrace {
+public class HubTraceSections {
```

Comment:
We could alternatively just call it `TraceSections` without reference to the HUB (which is the module that contains this class anyway.)